gnd/src/main/java/com/google/android/gnd/persistence/geojson/GeoJsonParser.java

```diff
@@ -42,34 +40,24 @@
 
   private static final String TAG = GeoJsonParser.class.getSimpleName();
   private final OfflineUuidGenerator uuidGenerator;
+  private static final String FEATURES_KEY = "features";
 
   @Inject
   GeoJsonParser(OfflineUuidGenerator uuidGenerator) {
     this.uuidGenerator = uuidGenerator;
   }
 
-  private String readJsonFile(File file) throws IOException {
-    InputStream is = new FileInputStream(file);
-    BufferedReader buf = new BufferedReader(new InputStreamReader(is));
-    StringBuilder sb = new StringBuilder();
-
-    int res = buf.read();
-    while (res != -1) {
-      sb.append(res);
-      res = buf.read();
-    }
-    return sb.toString();
-  }
-
   /**
    * Returns the immutable list of tiles specified in {@param geojson} that intersect {@param
    * bounds}.
    */
   public ImmutableList<Tile> intersectingTiles(LatLngBounds bounds, File file) {
     try {
-      String fileContents = readJsonFile(file);
+      String fileContents = FileUtils.readFileToString(file, Charset.forName("UTF-8"));
```

Comment:
Nit: Please extract Charset to constant.

Reply:
Done!